ci-operator/step-registry/openshift-qe/web-burner/openshift-qe-web-burner-ref.yaml

```diff
@@ -0,0 +1,74 @@
+ref:
+  as: openshift-qe-web-burner
+  from_image:
+    namespace: ci
+    name: ocp-qe-perfscale-ci
+    tag: latest
+  env:
+  - name: BAREMETAL
+    default: "true"
+    documentation: |-
+      Set to true if the job is for baremetal
+  - name: BFD
+    default: "false"
+    documentation: |-
+      Enable BFD
+  - name: CHURN
+    default: "false"
+    documentation: |-
+      Default is false, which means the workload will not delete and recreate a set number of namespaces
+  - name: CRD
+    default: "false"
+    documentation: |-
+      Enable the new AdminPolicyBasedExternalRoute CR
+  - name: E2E_VERSION
+    default: "v2.1.7"
```

Comment:
This needs an update to `v2.1.8` in order for the latest web-burner changes to get reflected.